.github/workflows/test-check.yaml

```diff
@@ -74,26 +70,16 @@ jobs:
       - name: "⚙️ Install dependencies"
         run: pip3 install .[dev]
       - name: "🔬 Running pytorch tests"
-        run: make test TARGETS=pytorch
+        run: |
+          pytest tests/llmcompressor/pytorch
```

Comment:
Could we run with -v?

Reply:
Yeah I usually add -s for verbose error messages if fail

Reply:
I usually add -s so it tells the error message if fail. 
-v applies to both success and fail?
If so it might cause longer run time

Reply:
I think -s will also show prints but -v doesnt always. I'm gonna land this for now and we can adjust if we aren't happy with the level of logging. Does that sound good?

Reply:
Yes